RELEASE.md

```diff
@@ -8,94 +8,83 @@ Versioning
 
 Follow [semantic versioning 2.0.0][semver] to choose the new version number.
 
-Prepare change log entries
```

Comment:
Release notes will be generated by release API automatically and the format of it doesn't look guaranteed. We only need to confirm that the output is correct regardless of the format.

Reply:
I fixed it in the similar manner as https://github.com/topolvm/pie/pull/90#discussion_r1364715431 .